system/doc/reference_manual/typespec.xml

```diff
@@ -622,6 +622,14 @@
         or not.
       </p>
     </note>
+    <note>
+      <p>Since <c>OTP-26.1</c>, argument names become global in the type specification.
+      That is, the following examples are all isomorphic:
```

Comment:
I think that more people find *equivalent* easier to understand than *isomorphic*.

```suggestion
      That is, the following examples are all equivalent:
```

Reply:
Is it also equivalent to `-spec id(X :: tuple()) -> X :: tuple()`.? If so, - adding this to the list of equivalent specs would add clarity and remove ambiguity.

Reply:
Yes, these would be equivalent